packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapInitializer.java

```diff
@@ -5,74 +5,51 @@
+import androidx.annotation.Nullable;
 import androidx.annotation.VisibleForTesting;
 import com.google.android.gms.maps.MapsInitializer;
 import com.google.android.gms.maps.MapsInitializer.Renderer;
 import com.google.android.gms.maps.OnMapsSdkInitializedCallback;
 import io.flutter.plugin.common.BinaryMessenger;
-import io.flutter.plugin.common.MethodCall;
-import io.flutter.plugin.common.MethodChannel;
 
 /** GoogleMaps initializer used to initialize the Google Maps SDK with preferred settings. */
 final class GoogleMapInitializer
-    implements OnMapsSdkInitializedCallback, MethodChannel.MethodCallHandler {
-  private final MethodChannel methodChannel;
+    implements OnMapsSdkInitializedCallback, Messages.MapsInitializerApi {
   private final Context context;
-  private static MethodChannel.Result initializationResult;
+  private static Messages.Result<Messages.PlatformRendererType> initializationResult;
   private boolean rendererInitialized = false;
 
   GoogleMapInitializer(Context context, BinaryMessenger binaryMessenger) {
     this.context = context;
 
-    methodChannel =
-        new MethodChannel(binaryMessenger, "plugins.flutter.dev/google_maps_android_initializer");
-    methodChannel.setMethodCallHandler(this);
+    Messages.MapsInitializerApi.setUp(binaryMessenger, this);
   }
 
   @Override
-  public void onMethodCall(MethodCall call, MethodChannel.Result result) {
-    switch (call.method) {
-      case "initializer#preferRenderer":
-        {
-          String preferredRenderer = (String) call.argument("value");
-          initializeWithPreferredRenderer(preferredRenderer, result);
-          break;
-        }
-      default:
-        result.notImplemented();
-    }
-  }
-
-  /**
-   * Initializes map renderer to with preferred renderer type. Renderer can be initialized only once
-   * per application context.
-   *
-   * <p>Supported renderer types are "latest", "legacy" and "default".
-   */
-  private void initializeWithPreferredRenderer(
-      String preferredRenderer, MethodChannel.Result result) {
+  public void initializeWithPreferredRenderer(
+      @Nullable Messages.PlatformRendererType type,
+      @NonNull Messages.Result<Messages.PlatformRendererType> result) {
     if (rendererInitialized || initializationResult != null) {
       result.error(
-          "Renderer already initialized", "Renderer initialization called multiple times", null);
+          new Messages.FlutterError(
+              "Renderer already initialized",
+              "Renderer initialization called multiple times",
+              null));
     } else {
       initializationResult = result;
-      switch (preferredRenderer) {
-        case "latest":
-          initializeWithRendererRequest(Renderer.LATEST);
-          break;
-        case "legacy":
-          initializeWithRendererRequest(Renderer.LEGACY);
-          break;
-        case "default":
-          initializeWithRendererRequest(null);
-          break;
-        default:
```

Comment:
The special "default" value no longer exists, because requesting the default is now expressed as `null` in the Pigeon API, just as it already was at both the Dart API and Maps SDK API levels.